### PR TITLE
Performance optimization

### PR DESCRIPTION
**Pull request by troydai**
1. Delay import platform
2. Delay import colorama

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-VERSION = '0.2.0'
+VERSION = '0.2.1'
```
Comment (derekbekoe) [inline setup.py]: No need to bump version. We create a release commit at release time.

File: knack/log.py [hunk 2]
```diff
@@ -35,19 +32,27 @@ def get_logger(module_name=None):
...
-        return wrap_msg_with_color
-
-    COLOR_MAP = {
-        logging.CRITICAL: _color_wrapper(colorama.Fore.RED),
-        logging.ERROR: _color_wrapper(colorama.Fore.RED),
-        logging.WARNING: _color_wrapper(colorama.Fore.YELLOW),
-        logging.INFO: _color_wrapper(colorama.Fore.GREEN),
-        logging.DEBUG: _color_wrapper(colorama.Fore.CYAN)
-    }
+        return cls.COLOR_MAP.get(level, None)
```
Comment (derekbekoe) [inline knack/log.py]: Can you add a test to `test_log.py` for this change?
As more and more products depend on knack, I want to make sure the test coverage is there for changes.

Comment (troydai) [reply knack/log.py]: ok.

**Comment by troydai** [comment]
@derekbekoe test added and rebased.

**Pull request merged**